Modules/CodeTrek/Routes/web.php

```diff
@@ -14,6 +14,7 @@
 Route::prefix('codetrek')->middleware('auth')->group(function () {
     Route::get('/', 'CodeTrekController@index')->name('codetrek.index');
     Route::post('/', 'CodeTrekController@store')->name('codetrek.store');
+    Route::post('/storeCodeTrekApplicantFeedback', 'CodeTrekController@storeCodeTrekApplicantFeedback')->name('codetrek.storeCodeTrekApplicantFeedback');
```

Comment:
The formatting of route name and URL is not correct
```suggestion
    Route::post('/store-feedback', 'CodeTrekController@storeCodeTrekApplicantFeedback')->name('codetrek.store-feedback');
```